examples/DigitalRain/DigitalRain.ino

```diff
@@ -29,10 +28,24 @@ KEYMAPS(
 
 // *INDENT-ON*
 
+KALEIDOSCOPE_INIT_PLUGINS(
+
+  // LEDControl provides support for other LED modes
+  LEDControl,
+
+  // We start with the LED effect that turns off all the LEDs.
+  LEDOff,
+
+  // Support for DigitalRain
+   LEDDigitalRainEffect
```

Comment:
Could you fix the indentation here please?

Reply:
Done.